juniper_official/Protocols/Mpls/check-rsvp-neighbor-state-oc.rule

```diff
@@ -0,0 +1,100 @@
+/*
+ * Monitors MPLS LSP state and notifies when LSP state is UP or DOWN.
+ * One inputs control detection
```

Comment:
And here..

Reply:
Changed to "one input controls detection"